src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithoutStrongNames.txt

```diff
@@ -155,8 +155,10 @@ Relative path: 'SonarLint.VisualStudio.ConnectedMode.dll'
 Referenced assemblies:
 - 'LibGit2Sharp, Version=0.26.0.0, Culture=neutral, PublicKeyToken=7cbde695407f0333'
 - 'Microsoft.Alm.Authentication, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null'
+- 'Microsoft.CodeAnalysis.Workspaces, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
```

Comment:
The `VSWorkspace` requires a couple of additional assemblies. These ship with VS so we don't need to add them to the VSIX..